repl/src/main/scala/ammonite/repl/Util.scala

```diff
@@ -195,6 +195,7 @@ object Util{
 
 object Timer{
   var current = 0L
+  var show = false
```

Comment:
Instead of using a global mutable flag, could we instead use a environment variable that we set from Bash? 

e.g. an `AMMONITE_LOG_TIMINGS` that we can get from `System.getenv`